src/modules/assertions/require_cc0.ts

```diff
@@ -0,0 +1,23 @@
+import { context, getOctokit } from "@actions/github";
+import { setFailed } from "@actions/core";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+import _ from "lodash";
+
+// BOT will fail if no CC0 detected in PR body
+// Just after: Verify correct environment and request context in main.ts, if no CC0 is detected, 
+// BOT will fail and comment: "error: No CC0 waiver detected" 
+
+export const  checkCC0 = () => {
+        
+	const github = getOctokit(GITHUB_TOKEN).rest;
+	if !((context.payload?.pull_request?.body).search("CC0")) {
```

Comment:
I'd recommend you check for the following instead
```suggestion
	if !(context.payload?.pull_request?.body.search("##Copyright\nCopyright and related rights waived via [CC0]")) {
```

This *should* counteract any issues with CC0 accidentally appearing in e.g. an included asset file. The specificity should now make this no longer an issue.